packs/st2ci/actions/mistral.yaml

```diff
@@ -8,11 +8,9 @@ parameters:
   test_host:
     type: string
     description: The name of the hosts where the integration test(s) will run.
-    default: None
```

Comment:
So what value is passed to test_host if this is removed. If this value is not provided (check for None), then mistral WF calls linux.dig otherwise it'll use the provided test_host.